### DISCUSSION TITLE
Using multiple colors in the section?

### DISCUSSION THREAD

**Discussion by c0h2r** [startup-nvim/startup.nvim#9] [Q&A] [answered]
Is it possible to either use several colors in the section or make several sections appear on the same "line"? For example, text and hotkeys in the `body` section of the `evil.lua` colorscheme, or something like one of DT's colorscripts in the header.
![image](https://user-images.githubusercontent.com/59478667/146026180-a5d7d66c-ca5f-405b-a7f1-bc88d32a46a3.png)

**Comment by max397574** [comment]
can you send me a link to that?

**Reply by c0h2r** [reply]
You mean the script? If so, here's [the gitlab repo](https://gitlab.com/dwt1/shell-color-scripts).

**Comment by max397574** [comment, marked as answer]
at the moment it isn't supported
it would be possible but it makes everything much more complicated
I might implement it in the future

**Reply by c0h2r** [reply]
Oh, okay. Thanks for the answer.

**Comment by max397574** [comment]
you could use a plugin like this
https://github.com/norcalli/nvim-terminal.lua
which allows the use of color codes

**Reply by c0h2r** [reply]
Thanks, but I've already done it a bit differently:
![2021-12-14-224507_1920x1080_scrot](https://user-images.githubusercontent.com/59478667/146068663-ebdd2c04-a7e7-4592-a629-409074e09271.png)

**Reply by j-steinbach** [reply]
How did you do it? I would guess different sections with different colors?

**Reply by c0h2r** [reply]
> How did you do it? I would guess different sections with different colors?

Yeah, exactly

```
  textbox = {
    type = "text",
    align = "center",
    fold_section = false,
    title = "space invimders",
    margin = 5,
    content = {
      "                               ",
      "                               ",
      "      ╭─╴ ╭─┐ ╭─┐ ╭─╴ ╭─╴      ",
      "      ╰─╮ │─╯ │─│ │   │─       ",
      "      ╶─╯ ╵   ╵ ╵ └─╴ └─╴      ",
      "╷ ╷ ╷ ╷ ╷ ╷ ╭┬┐ ┌─╮ ╭─╴ ┌─╮ ╭─╴",
      "│ │╲│ │╭╯ │ │││ │ │ │─  │─╯ ╰─╮",
      "╵ ╵ ╵ └╯  ╵ ╵ ╵ ╵─╯ └─╴ ╵ ╵ ╶─╯",
  }
    ,
    highlight = "GruvboxYellow",
    default_color = "#fbf1c7",
    oldfiles_amount = 0,
  },
  
  header = {
    type = "text",
    align = "center",
    fold_section = false,
    title = "Header",
    margin = 5,
    content = {
      "          ▄ ▀▄   ▄▀ ▄          ",
      "          █▄█▀███▀█▄█          ",
      "          ▀█████████▀          ",
      "           ▄▀     ▀▄           ",
  }
    ,
    highlight = "GruvboxPurple",
    default_color = "",
    oldfiles_amount = 0,
  },

  header1 = {
    type = "text",
    align = "center",
    fold_section = false,
    title = "Header1",
    margin = 4,
    content = {
      "   ▄██▄                 ▄██▄   ",
      " ▄█▀██▀█▄             ▄█▀██▀█▄ ",
      " ▀█▀██▀█▀             ▀█▀██▀█▀ ",
      " ▀▄    ▄▀             ▀▄    ▄▀ ",
      "                               ",  }
    ,
    highlight = "GruvboxBlue",
    default_color = "",
    oldfiles_amount = 0,
  }
```
